mtucker01752-boston.md

```diff
@@ -0,0 +1,7 @@
+
+## Boston, MA
+
+### Places to eat
+- Tavern in the Square - http://taverninthesquare.com
```

Comment:
What about to use markdown for links like?:

- [Tavern in the Square](http://taverninthesquare.com)

What do you think?